idea/src/org/jetbrains/kotlin/idea/codeInsight/surroundWith/expression/KotlinExpressionSurroundDescriptor.java

```diff
@@ -33,7 +33,11 @@
             new KotlinWhenSurrounder() ,
             new KotlinRuntimeTypeCastSurrounder(),
             new KotlinWithIfExpressionSurrounder(/* withElse = */false),
-            new KotlinWithIfExpressionSurrounder(/* withElse = */true)
+            new KotlinWithIfExpressionSurrounder(/* withElse = */true),
+            new KotlinTryCatchExpressionSurrounder(),
+            new KotlinTryCatchFinallyExpressionSurrounder(),
+            new KotlinIfElseExpressionSurrounder(false),
```

Comment:
I am not a decisive person, not Kotlin team member, but I think that you should follow an example above - I believe that was made for a specific purpose - and add commented parameter to `KotlinIfElseExpressionSurrounder` callings.

So your code would be:
```
   new KotlinWithIfExpressionSurrounder(/* withElse = */true),
            new KotlinTryCatchExpressionSurrounder(),
            new KotlinTryCatchFinallyExpressionSurrounder(),
            new KotlinIfElseExpressionSurrounder(/* withBraces = */false),
            new KotlinIfElseExpressionSurrounder(/* withBraces = */true)